content/how-to/dedicated-hosting/create-new-user-on-active-directory-domain/index.md

```diff
@@ -19,33 +19,36 @@ Active Directory user.
 1. Log in to the [MyRackspace portal](https://login.rackspace.com/login) with
    your username and password.
 
-2. In the top navigation bar, choose **Select a Product** > **Rackspace Dedicated**.
+   <img width="409" alt="newuser3" src="/support/how-to/create-new-user-on-active-directory-domain/newuser3.png">
```

Comment:
the alt tage should be a few words describing what is happening in the screenshot. That way the browser reads it to people with vision problems.

Reply:
modified the alt text as "enter user name and password"